lib/activerecord-multi-tenant/sidekiq.rb

```diff
@@ -33,6 +33,23 @@ def call(worker_class, msg, queue)
   end
 end
 
+if defined?(Sidekiq)
```

Comment:
Wouldn't this always be true since it'd be defined by the declaration directly above?

Regardless, I'm torn. Maybe I'm old hat, but I don't like required files setting up hooks. A `Sidekiq::Middleware::MultiTenant.register` method would allow the consumer to opt-in

Reply:
Yeah, I think that is a bit redundant. I was proofing against none sidekiq setup, but thats OK since the module is defined anyways. 

re: hooks, I am happy with either. This comment made me think that the expectation is to just require the file and that should be all. Which is opt-in as well I guess, and not bad if the gem takes care of it. Plus, the operation is idempotent. 

https://github.com/citusdata/activerecord-multi-tenant/issues/77#issuecomment-579089498

Again, happy to introduce a function too.